### PR TITLE
REGEX - More stores

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: data/ocr/regex_stores.txt [hunk 1]
```diff
@@ -6,6 +6,9 @@ Carrefour
 Carrefour||grand jury
 Colruyt
 Delhaize
+Edeka
+Eroski
+Hacendado
```
Comment (raphael0202) [inline data/ocr/regex_stores.txt]: Isn't Hacendado a brand from Mercadona? You should write Mercadona||Hacendado if so

File: data/ocr/regex_stores.txt [hunk 2]
```diff
@@ -14,4 +21,109 @@ Magasins U||u saveurs
...
+Leclerc||confis[ée]o
+Leclerc||copains copines
+Leclerc||day break
+Leclerc||d[ée]li d[ée]lo
+Leclerc||d[ée]lisse
+Leclerc||d[ée]li'light
+Leclerc||douceur du verger
+Leclerc||edulcorel
+Leclerc||entre'aide
+Leclerc||[ÉE]quador
```
Comment (teolemon) [inline data/ocr/regex_stores.txt]:
```suggestion
Leclerc||[ée]quador
```

File: data/ocr/regex_stores.txt [hunk 3]
```diff
@@ -14,4 +21,109 @@ Magasins U||u saveurs
...
+Leclerc||brin de jour
+Leclerc||britley
+Leclerc||chaque jour r[ée]duit en lactose
+Leclerc||chaque jour sans gluten
+Leclerc||chantegril
+Leclerc||comptoir du grain
+Leclerc||côt[ée]' snack
+Leclerc||côt[ée] snack
+Leclerc||côt[ée] table
+Leclerc||couleurs Vives
```
Comment (teolemon) [inline data/ocr/regex_stores.txt]:
```suggestion
Leclerc||couleurs vives
```

File: data/ocr/regex_stores.txt [hunk 4]
```diff
@@ -14,4 +21,109 @@ Magasins U||u saveurs
...
+Leclerc||fruistar
+Leclerc||jafaden
+Leclerc||jean's
+Leclerc||hilb[öo]rg
+Leclerc||laqueuille
+Leclerc||lagoa
+Leclerc||les caract[èe]res
+Leclerc||les crois[ée]s
+Leclerc||les go[ëe]lleries
+Leclerc||l’Origine du goût
```
Comment (teolemon) [inline data/ocr/regex_stores.txt]:
```suggestion
Leclerc||l’origine du goût
```

File: data/ocr/regex_stores.txt [hunk 5]
```diff
@@ -14,4 +21,109 @@ Magasins U||u saveurs
...
+Leclerc||mots d'enfants
+Leclerc||notre jardin
+Leclerc||nustikao
+Leclerc||nadya
+Leclerc||nat & vie
+Leclerc||nat&vie
+Leclerc||nat et vie
+Leclerc||new’r
+Leclerc||new’ r
+Leclerc||nid d'Abeille
```
Comment (teolemon) [inline data/ocr/regex_stores.txt]:
```suggestion
Leclerc||nid d'abeille
```

File: data/ocr/regex_stores.txt [hunk 6]
```diff
@@ -14,4 +21,109 @@ Magasins U||u saveurs
...
+Leclerc||Œufs de nos r[ée]gions
```
Comment (teolemon) [inline data/ocr/regex_stores.txt]:
```suggestion
Leclerc||œufs de nos r[ée]gions
```

File: data/ocr/regex_stores.txt [hunk 7]
```diff
@@ -14,4 +21,109 @@ Magasins U||u saveurs
...
+Leclerc||st di[ée]ry
+Leclerc||saint charmin
+Leclerc||smicy
+Leclerc||tables du monde
+Leclerc||tablette d’or
+Leclerc||tablier blanc
+Leclerc||t[ée]va
+Leclerc||tis[ée]a
+Leclerc||tokapi
+Leclerc||tradilège
```
Comment (teolemon) [inline data/ocr/regex_stores.txt]:
```suggestion
Leclerc||tradil[èe]ge
```

**Comment by teolemon** [comment]
Fixed for Mercadona @raphael0202

File: data/ocr/regex_stores.txt [hunk 8]
```diff
@@ -4,8 +4,15 @@ Auchan
 Biocoop
 Carrefour
 Carrefour||grand jury
+Casino
 Colruyt
+Coop
```
Comment (raphael0202) [inline data/ocr/regex_stores.txt]: @teolemon Don't you think coop can lead to many false positive?

File: data/ocr/regex_stores.txt [hunk 9]
```diff
@@ -14,4 +21,162 @@ Magasins U||u saveurs
...
-Sainsbury's
\ No newline at end of file
+Netto
+Picard
+REWE
+Sainsbury's
+Tesco
+Waitrose
+Intermarché
+Intermarché||paturages
```
Comment (raphael0202) [inline data/ocr/regex_stores.txt]: faux positifs ?

**Comment by raphael0202** [comment]
As the PR was quite old, I've integrated your changes manually in 763f902207ea7aafaba9dd812c9c14115e6b2a18. Thanks!

**Pull request closed without merging**